features/org.jboss.tools.arquillian.feature/feature.xml

```diff
@@ -14,7 +16,7 @@
       %copyright
    </copyright>
 
-   <license url="%licenseURL">
+   <license>
```

Comment:
shouldn't you remove the whole license node? given %license can not be resolved anymore...